app/src/main/java/io/intrepid/contest/models/Contest.java

```diff
@@ -82,6 +86,14 @@ public void setEntries(List<Entry> entries) {
         this.entries = entries;
     }
 
+    public void setEndedAt(String endedAt) {
```

Comment:
The setters were added solely for the MockApi to return a ContestWrapper response that has ended after ending a contest. Perhaps we need a test to check if after calling endContest() from our mockApi that the contest has a submissionEndedDate and contestEndedDate. A bit out of scope for this ticket though. thoughts?